test/python/expected/upgrade_validation/13_5/expected_dependency.out

```diff
@@ -0,0 +1,1189 @@
+Could not find dependencies on function sys.varchar2int4(sys."varchar")
+Could not find dependencies on function sys.varchar2int8(sys."varchar")
+Could not find dependencies on function sys.varchar2smalldatetime(character varying)
+Could not find dependencies on function sys.varchar2smalldatetime(sys."varchar")
+Could not find dependencies on function sys.varchar2uniqueidentifier(character varying,integer,boolean)
+Could not find dependencies on function sys.varchar2uniqueidentifier(sys."varchar",integer,boolean)
+Could not find dependencies on function sys.varchar_sqlvariant(character varying)
+Could not find dependencies on function sys.varchar_sqlvariant(sys."varchar")
+Could not find dependencies on function sys.varchar_to_name(character varying)
+Could not find dependencies on function sys.varchar_to_name(sys."varchar")
+Could not find dependencies on function sys.varcharbinary(character varying,integer,boolean)
+Could not find dependencies on function sys.varcharbinary(sys."varchar",integer,boolean)
+Could not find dependencies on function sys.varcharcmp(sys."varchar",sys."varchar")
+Could not find dependencies on function sys.varchareq(sys."varchar",sys."varchar")
+Could not find dependencies on function sys.varcharge(sys."varchar",sys."varchar")
+Could not find dependencies on function sys.varchargt(sys."varchar",sys."varchar")
+Could not find dependencies on function sys.varcharin(cstring)
+Could not find dependencies on function sys.varcharle(sys."varchar",sys."varchar")
+Could not find dependencies on function sys.varcharlt(sys."varchar",sys."varchar")
+Could not find dependencies on function sys.varcharne(sys."varchar",sys."varchar")
+Could not find dependencies on function sys.varcharout(sys."varchar")
+Could not find dependencies on function sys.varcharrecv(internal)
+Could not find dependencies on function sys.varcharsend(sys."varchar")
+Could not find dependencies on function sys.varcharvarbinary(character varying,integer,boolean)
+Could not find dependencies on function sys.varcharvarbinary(sys."varchar",integer,boolean)
+Could not find dependencies on function sys.version()
+Could not find dependencies on function sys.xact_state()
+Could not find dependencies on function sys.year(anyelement)
+Could not find dependencies on operator <(sys."varchar",sys."varchar")
+Could not find dependencies on operator <(sys.bpchar,sys.bpchar)
+Could not find dependencies on operator <=(sys."varchar",sys."varchar")
+Could not find dependencies on operator <=(sys.bpchar,sys.bpchar)
+Could not find dependencies on operator <>(sys."varchar",sys."varchar")
+Could not find dependencies on operator <>(sys.bpchar,sys.bpchar)
+Could not find dependencies on operator <>(sys.bpchar,text)
+Could not find dependencies on operator <>(text,sys.bpchar)
+Could not find dependencies on operator =(sys."varchar",sys."varchar")
+Could not find dependencies on operator =(sys.bpchar,sys.bpchar)
+Could not find dependencies on operator =(sys.bpchar,text)
```

Comment:
Do we really need these dependencies on operators and collations in the expected output file?

Reply:
Yeah, we have already seen issues related to operators in prod when customer objects depend upon sys operators. Similarly we should also have minimal dependencies on collations as a table columns so that we atleast be testing each collation in upgrade.

Reply:
Sounds good!